app/component/map/tile-layer/BikeRentalStations.js

```diff
@@ -89,6 +89,7 @@ class BikeRentalStations {
               !this.features.some(feature =>
                 this.shouldShowStation(
                   feature.properties.id,
+                  feature.properties.name,
```

Comment:
This is not needed if we use id instead of name (just be sure that the id is actually in the same format as the one you pass as sometimes it's feedscoped and sometimes isnt.